gammapy/datasets/core.py

```diff
@@ -19,11 +19,11 @@
 
 class Dataset(abc.ABC):
     """Dataset abstract base class.
-    For now, see existing examples of type of datasets:
+    For now, see existing examples of type of datasets :
```

Comment:
Here I'm not sure that we should put a space before colon, because it is really an english sentence and not a "parameters colon". 

```suggestion
    For now, see existing examples of type of datasets:
```